mdx-models/src/main/java/com/mx/path/model/mdx/accessor/feature/Feature.java

```diff
@@ -0,0 +1,31 @@
+package com.mx.path.model.mdx.accessor.feature;
+
+/**
+ * Enum representing different features in the application.
+ * <p>
+ * This enum serves as a centralized definition of features available in
+ * the application. Each constant corresponds to a specific feature,
+ * allowing for type-safe handling and easy reference throughout the codebase.
+ * </p>
+ * Usage:
+ * <p>
+ * Use these feature constants when implementing feature-specific logic,
+ * error handling, or when setting headers for error responses.
+ * </p>
+ */
+public enum Feature {
```

Comment:
We have a feature enum in core. Do we need one here as well?

Reply:
@tessstoddard  Thanks for letting me know. I changed the logic to incorporate the existing enum instead of creating a new one.